application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -52,7 +52,7 @@
      */
     public TeXCommand() {
         super("tex",
-                "This command accepts a latex expression and generates an image corresponding to it.",
+                "This command accepts a regular and an inline latex expression and generates an image corresponding to it.",
```

Comment:
Not your fault, but I honestly dont like this description. It feels like it is more written for a developer instead of an actual average Discord user.

My proposal:
```java
"Renders LaTeX, also supports inline $-regions like 'see this $\frac[x}{2\$'."
```

Reply:
Okay but honestly, it's not the average Discord user.

How is the average Discord user going to use a LaTeX command if they don't know what LaTeX is?

Reply:
Thats fair. But the description is really rather the opposite of compact. And it doesnt explain what an inline-latex is.